src/App.js

```diff
@@ -8,8 +8,6 @@ const Users = lazy(() => import("./pages/users/Users"));
 const ErrorPage = lazy(() => import("./pages/ErrorPage"));
 const Dashboard = lazy(() => import("./pages/dashboard/Dashboard"));
 
-
-
 export default function App() {
   return (
     <Router>
```

Comment:
please use translations on loading page message

Reply:
added translation for loading page message